app/src/app/translation.initializer.ts

```diff
@@ -0,0 +1,21 @@
+import { catchError } from 'rxjs/operators';
+import { empty, EMPTY } from 'rxjs';
+import { TranslateService } from '@ngx-translate/core';
+import { EnplugService } from './enplug.service';
+
+export function translationInitializer(enplug: EnplugService, translate: TranslateService) {
+  return () => new Promise<void>(async (resolve, reject) => {
+    const settings = await enplug.settings.all as any;
+    const locale = settings.locale || 'en';
```

Comment:
This default should come from the web player. If it isn't already, please file a ticket and remove 'en'.